src/PufferVaultMainnet.sol

```diff
@@ -189,6 +228,8 @@ contract PufferVaultMainnet is PufferVault {
 
     /**
      * @notice Allows the `msg.sender` to burn his shares
+     * @dev Restricted to PufferProtocol smart contract
+     * We use it to burn the bond if the node operator gets slashed
      * @param shares The amount of shares to burn
      */
     function burn(uint256 shares) public restricted {
```

Comment:
Nbd, but we mentioned we could allow anyone to call this to basically make a donation. The alternative they'd have is to send to something like address(0), but doesn't really burn the token / change the exchange ratio. IIRC we discussed making this pausable

Reply:
Less things to think about. If somebody wants to donate, they can do so by depositing ETH/stETH/WETH